pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
   "PyYAML>=6.0",
   "requests>=2.31.0",
   "rich>=13.5.2,<14",
-  "httpx>=0.27.0"
+  "httpx>=0.28.0"
```

Comment:
I don't think we need to bump here right?

Reply:
True